Modules/QuestieReputation.lua

```diff
@@ -1,10 +1,13 @@
 ---@class QuestieReputation
+------@field hasFactionChange boolean @true if some faction leveled up since the last QuestieQuest:CalculateAndDrawAvailableQuestsIterative
```

Comment:
```suggestion
---@field hasFactionChange boolean @true if some faction leveled up since the last QuestieQuest:CalculateAndDrawAvailableQuestsIterative
```